<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/x2go-bugs/2011-March/index.html" >
   <LINK REL="made" HREF="mailto:x2go-bugs%40lists.berlios.de?Subject=Re%3A%20%5BX2go-bugs%5D%20%5BTickets%20%2348%5D%20Re%3A%20pyhoca-gui%20does%20not%20start%20on%20Windows&In-Reply-To=%3C20110302134208.Horde.xl7DQoeoSH1NbjsgwcOQqjA%40code.x2go.org%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000115.html">
   <LINK REL="Next"  HREF="000117.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows</H1>
    <B>bugs at x2go.org</B> 
    <A HREF="mailto:x2go-bugs%40lists.berlios.de?Subject=Re%3A%20%5BX2go-bugs%5D%20%5BTickets%20%2348%5D%20Re%3A%20pyhoca-gui%20does%20not%20start%20on%20Windows&In-Reply-To=%3C20110302134208.Horde.xl7DQoeoSH1NbjsgwcOQqjA%40code.x2go.org%3E"
       TITLE="[X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows">bugs at x2go.org
       </A><BR>
    <I>Wed Mar  2 13:42:08 CET 2011</I>
    <P><UL>
        <LI>Previous message: <A HREF="000115.html">[X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows
</A></li>
        <LI>Next message: <A HREF="000117.html">[X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#116">[ date ]</a>
              <a href="thread.html#116">[ thread ]</a>
              <a href="subject.html#116">[ subject ]</a>
              <a href="author.html#116">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Ticket URL: <A HREF="http://code.x2go.org/horde4/whups/ticket/?id=48">http://code.x2go.org/horde4/whups/ticket/?id=48</A>
------------------------------------------------------------------------------
  Ticket             | 48
  Updated By         | <A HREF="https://lists.berlios.de/mailman/listinfo/x2go-bugs">dick.kniep at lindix.nl</A>
  Summary            | pyhoca-gui does not start on Windows
  Queue              | pyhoca-gui
  Version            | HEAD
  Type               | Bug
  State              | Feedback
  Priority           | 3. High
  Operating System   | 7
  Owners             | Mike Gabriel
------------------------------------------------------------------------------


<A HREF="https://lists.berlios.de/mailman/listinfo/x2go-bugs">dick.kniep at lindix.nl</A> (2011-03-02 13:42) wrote:

NXPROXY - Version 3.4.0

Copyright (C) 2001, 2010 NoMachine.
See <A HREF="http://www.nomachine.com/">http://www.nomachine.com/</A> for more information.

Info: Proxy running in client mode with pid '3444'.
Session: Starting session at 'Wed Mar  2 13:39:18 2011'.
Info: Connecting to remote host 'localhost:30004'.
Info: Connection to remote proxy 'localhost:30004' established.
Info: Connection with remote proxy completed.
Warning: Unrecognized session type 'unix-kde-depth_24'. Assuming agent  
session.
Warning: Failed to read data from the X auth command.
Warning: Generated a fake cookie for X authentication.
Info: Using ADSL link parameters 512/24/1/0.
Info: Using cache parameters 4/4096KB/8192KB/8192KB.
Info: Using pack method '16m-jpeg-9' with session 'unix-kde-depth_24'.
Info: Using ZLIB data compression 1/1/32.
Info: Using ZLIB stream compression 4/4.
Info: No suitable cache file found.
Hi Mike,

Here is the log.

Info: Forwarding X11 connections to display 'localhost:0'.
Session: Session started at 'Wed Mar  2 13:39:20 2011'.
Error: Connection to 'localhost:0' failed. Error is 111 'Connection refused'.
Session: Terminating session at 'Wed Mar  2 13:39:34 2011'.
Info: Your session was closed before reaching a usable state.
Info: This can be due to the local X server refusing access to the client.
Info: Please check authorization provided by the remote X application.
Session: Session terminated at 'Wed Mar  2 13:39:34 2011'.

It seems that the X-server simply refuses the connection. Probably we  
need to add an xauth to the installation script!!






</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000115.html">[X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows
</A></li>
	<LI>Next message: <A HREF="000117.html">[X2go-bugs] [Tickets #48] Re: pyhoca-gui does not start on Windows
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#116">[ date ]</a>
              <a href="thread.html#116">[ thread ]</a>
              <a href="subject.html#116">[ subject ]</a>
              <a href="author.html#116">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/x2go-bugs">More information about the X2go-Bugs
mailing list</a><br>
</body></html>
